src/core/core.layouts.js

```diff
@@ -375,7 +376,9 @@ module.exports = {
 			left: chartArea.left,
 			top: chartArea.top,
 			right: chartArea.left + chartArea.w,
-			bottom: chartArea.top + chartArea.h
+			bottom: chartArea.top + chartArea.h,
+			height: CharacterData.h,
```

Comment:
`CharacterData`?

Reply:
Meant to type `chartArea` and I guess autocomplete had other ideas